pkgs/development/python-modules/json-home-client/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+# build inputs
+, typing-extensions
+, uri-template
+}:
+
+buildPythonPackage rec {
+  pname = "json_home_client";
+  version = "1.1.1";
+
+  disabled = pythonOlder "3.6";
+
+  #TODO: Figure out why Pypi doesn't host the source package
+  src = fetchFromGitHub {
+    owner = "plinss";
+    repo = pname;
+    rev = "v${version}";
+    sha512 = "sha512-ZmAWoOu+/SLTLaFDpNoxOeoHOx8tQhsuo+uqyZKXvRaSrJugBhvzH5WkPJ09SCVqSYtLcR2CVAE0vWjzrBJUXw==";
+  };
+
+  postPatch = ''
+    sed -i -e 's/0.0.0/${version}/' setup.py
+  '';
+
+  propagatedBuildInputs = [
+    typing-extensions
+    uri-template
+  ];
+
+  pythonImportsCheck = [ "${pname}" ];
```

```suggestion
  pythonImportsCheck = [ "json_home_client" ];
```
normalisation normally breaks this.